### DISCUSSION TITLE
Table headers.

### DISCUSSION THREAD

**Discussion by vinduajamesh** [TanStack/table#3957] [Q&A] [answered]
Do we need to call the table.createGroup even if I just have non-grouped headers. All the examples in the documentation uses this method. I'm just looking to build a simple table with a single row of headers. Thank you for the clarification!

**Comment by tannerlinsley** [comment, marked as answer]
Nope! The top level array is already a “group”

**Reply by vinduajamesh** [reply]
Thanks for clarifying!

**Reply by talatkuyuk** [reply]
This means that you can create the columns (data column or display column) in an array **without creating a group**. You don't need to create group, the react-table handles that columns you created in a top-level group in behind.

**Reply by matrs** [reply]
Sadly, `table.createGroup` isn't part of the API documentation for neither V7 nor V8. I got here looking for that information
